Shared/Tests.Shared/TestHelpers.cs

```diff
@@ -86,7 +86,7 @@ public static void RunEventLoop(int milliseconds)
 
         public static void RunEventLoop()
         {
-            RunEventLoop(1);
+            RunEventLoop(100);
```

Comment:
I rebased against `master` and got this commit in, somehow. It's unrelated to the PR.